article/app/controllers/LiveBlogController.scala

```diff
@@ -111,15 +110,14 @@ class LiveBlogController(
             Future.successful(common.renderHtml(MinuteHtmlPage.html(minute), minute))
           case (blog: LiveBlogPage, HtmlFormat) =>
             val dcrCouldRender = LiveBlogController.checkIfSupported(blog)
-            val participatingInTest = ActiveExperiments.isParticipating(LiveblogRendering)
             val isRecent = !LiveBlogController.isNotRecent(blog)
             val theme = blog.article.content.metadata.format.getOrElse(ContentFormat.defaultContentFormat).theme
             val design = blog.article.content.metadata.format.getOrElse(ContentFormat.defaultContentFormat).design
             val display = blog.article.content.metadata.format.getOrElse(ContentFormat.defaultContentFormat).display
             val isDeadBlog = LiveBlogController.isDeadBlog(blog)
             val properties =
               Map(
-                "participatingInTest" -> participatingInTest.toString,
+                "participatingInTest" -> "false",
```

Comment:
Can I just remove this parameter or are there dashboards that rely on it?

Reply:
🤷  probably best to remove in a subsequent PR?